src/Thermodynamics.jl

```diff
@@ -50,6 +50,9 @@ using KernelAbstractions: @print
 using CLIMAParameters: AbstractParameterSet
 using CLIMAParameters.Planet
 const APS = AbstractParameterSet
+entropy_dry_air(::APS) = 6864.8 # J / (kg K)
+entropy_water_vapor(::APS) = 10513.6 # J / (kg K)
+standard_temperature(::APS) = 298.15 # K
```

Comment:
These need to be moved to CLIMAParameters.jl, waiting on review / merging of [CLIMAParameter.jl's #51](https://github.com/CliMA/CLIMAParameters.jl/pull/51).